_includes/content/entityrule.md

```diff
@@ -1,13 +1,30 @@
+Entity rules specify the attributes of the source or destination of a packet that must match for the rule as a whole 
+to match.  Packets are matched on their IP and ports, if the rule contains multiple match criteria (for example, an 
```

Comment:
nits:
- I'd say "IPs and ports"
- I'd change the comma to a full stop and then start a new sentence.